user-interface/0001-WALL-wallet.md

```diff
@@ -0,0 +1,186 @@
+# Wallet
+A Vega wallet is required to prepare and submit transaction on Vega  (place, cancel, orders etc). See the [wallet docs](https://docs.vega.xyz/docs/mainnet/concepts/vega-wallet) for more on how "crypto" wallets work. 
+
+A wallet can contain many public/private key pairs. The public part of each key pair is known the [Party](../protocol/0017-PART-party.md) sometimes just referred to as a key or public key. 
+
+The primary job(s) of a wallet app is to [sign/encrypt transaction](../protocol/0022-AUTH-auth.md) (so the network can be sure they were sent a given party) and to broadcast these transactions to a node on the network.
+
+## Get wallet
+When on the wallet page of Vega.xyz, I...
+
+- **must** see links to the latest version of all the available wallets (e.g. desktop, command line and browser plugin) (inc github repos) (<a name="0001-WALL-001" href="#0001-WALL-001">0001-WALL-001</a>)
+- **must** see a primary download button is configured for the latest version and the operating system I am using. With a link to download for other operating systems too (<a name="0001-WALL-002" href="#0001-WALL-002">0001-WALL-002</a>)
+- **must** be able to use the default wallet version linked/downloadable from this page with the version of Vega currently running on mainnet (<a name="0001-WALL-076" href="#0001-WALL-076">0001-WALL-076</a>)
+
+...so I can download and run the wallet app on my machine and use it to interact with the Vega mainnet
+
+## Set up wallet / Restore wallet
+When opening the wallet for the first time, I...
+
+- **must** be prompted to opt into (or stay out of) analytics (<a name="0001-WALL-003" href="#0001-WALL-003">0001-WALL-003</a>)
+- **must** be able to restore a wallet from a seed phrase (<a name="0001-WALL-004" href="#0001-WALL-004">0001-WALL-004</a>)
+- **must** be able to create a new wallet (<a name="0001-WALL-005" href="#0001-WALL-005">0001-WALL-005</a>)
+  - **must** be shown the back up phrase (<a name="0001-WALL-006" href="#0001-WALL-006">0001-WALL-006</a>)
+  - **must** shown version number of the algorithm used to derive key pairs from the back up phrase (<a name="0001-WALL-007" href="#0001-WALL-007">0001-WALL-007</a>)
```

Comment:
Is this a must? Pretty sure the algo version is likely to not change. Also almost all users might not care about this and it has no bearing on their user journey?

Reply:
I assumed it is a must because it exists. If they used V1 to generate keys from seed, then restored a wallet with that same seed using V2 they will end up with new keys.
@ValentinTrinque @mattrussell36 I'm happy to say it is not required if you can tell me that there is no-one using V1?

Reply:
It is required, and we will have to support this forever sadly.